Assets/Scripts/Inventories/ItemDropper.cs

```diff
@@ -57,30 +57,30 @@ private struct DropRecord
             public int number;
         }
 
-        object ISaveable.CaptureState()
-        {
-            RemoveDestroyedDrops();
-            var droppedItemsList = new DropRecord[droppedItems.Count];
-            for (int i = 0; i < droppedItemsList.Length; i++)
-            {
-                droppedItemsList[i].itemID = droppedItems[i].GetItem().GetItemID();
-                droppedItemsList[i].position = new SerializableVector3(droppedItems[i].transform.position);
-                droppedItemsList[i].number = droppedItems[i].GetNumber();
-            }
-            return droppedItemsList;
-        }
+        // object ISaveable.CaptureState()
+        // {
+        //     RemoveDestroyedDrops();
+        //     var droppedItemsList = new DropRecord[droppedItems.Count];
+        //     for (int i = 0; i < droppedItemsList.Length; i++)
+        //     {
+        //         droppedItemsList[i].itemID = droppedItems[i].GetItem().GetItemID();
+        //         droppedItemsList[i].position = new SerializableVector3(droppedItems[i].transform.position);
+        //         droppedItemsList[i].number = droppedItems[i].GetNumber();
+        //     }
+        //     return droppedItemsList;
+        // }
```

Comment:
Do we still need these?

Reply:
Probably not. This is here in case we want to save items that may have been dropped, but It will probably be removed.